inginious/frontend/plugins/multilang/problems/code_multiple_languages_problem.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+from inginious.common.tasks_problems import CodeProblem
+from inginious.frontend.task_problems import DisplayableCodeProblem
+
+path_to_plugin = os.path.abspath(os.path.dirname(__file__))
+python_tutor_url = "http://127.0.0.1:8003/"
```

Comment:
The same for these constants, the name should be capitalized as it is more understandable that is a constant